cypress/integration/newUserForm.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Login', () => {
+  const app_base_url = Cypress.env('app_base_url')
+  beforeEach(() => {
+    cy.svcClientLogout()
+    cy.svcClientLogin().as('tokens')
+    cy.get('@tokens').then(cy.svcClientSetCookie)
+  })
+
+
+  it('Signs in, and shows privacy message on first login', () => {
+    cy.visit(app_base_url + '/home')
+    cy.get('div.ui.small.modal.transition.visible.active').invoke('show')
```

Comment:
Just a tip, we should try to avoid selecting elements by specific selectors like this. That helps to prevent our tests from breaking when changes are made to the DOM structure. Instead, we should try to write the tests from the perspective of an end user. To achieve that, we can use a helper library like [`cypress-testing-library`](https://github.com/testing-library/cypress-testing-library) that augments cypress's functionality with some special queries that make this easier. For example, this is how we may test the modal appears, and then disappears when you click the "Continue to" button:
```js
cy.findByText('Privacy Information').should('exist')
cy.findByText('Continue to MyRangeBC').click()
cy.findByText('Privacy Information').should('not.exist')
```

Reply:
Killer tips and thanks : ).  Trying out that lib now.